modules/speechInteraction/modules/googleSpeech/main.cpp

```diff
@@ -410,6 +406,163 @@ class Module : public yarp::os::RFModule, public googleSpeech_IDL
 };
 
 /********************************************************/
+<<<<<<< Updated upstream
```

Comment:
It seems there have been problems merging this file.

Reply:
I must have missed the line when merging locally, I will double check if the speech modules compile fine.

Reply:
Speech modules have been reverted to master, they need to be kept untouched in this pr.